docs/source/contributing/branch-policy.md

```diff
@@ -33,14 +33,20 @@ legacy
     When opening a pull request, the contributor must open it against `main`.
     If the pull request is a feature or a bugfix, and if the release manager deems it useful to the latest version's branch, they may ask the contributor to backport it to that branch.
 
+`17.x.x`
+:   At the moment of this writing, `17.x.x` is the current stable branch in git.
+    Upon the final release of version `18.0.0`, the `17.x.x` branch line will become legacy.
+    This is the current actively developed branch in git, meaning that it may receive new features and bug fixes.
+    Any new feature will be released generally to the `main` branch, and only backported if the Volto Team approves it.
+
 `16.x.x`
-:   This is the current actively developed branch in git, meaning that it may receive new features and bug fixes.
-    Its version is currently at 16.0.0-rc.1 as a release candidate.
-    It will become the stable version upon the final release of version 16.0.0.
+:   This is the LTS version of Volto, aligned with Plone 6.0 release.
```

Comment:
See https://github.com/plone/volto/pull/4905 and https://github.com/plone/volto/issues/4904

Reply:
We also talked about this one... although is controversial, we decided to stick with the denomination not as a LTS per se, but to align with the Plone LTS (6.0).

Reply:
I think it would be clearer to say this, and avoid "LTS".

```suggestion
:   This version of Volto has the [same maintenance and support schedule as Plone 6.0](https://plone.org/download/release-schedule).
```